src/csharp/Grpc.Tools/build/_grpc/Grpc.CSharp.xml

```diff
@@ -26,5 +26,16 @@
       </EnumProperty.DataSource>
     </EnumProperty>
 
+    <EnumProperty Name="ClientBaseType" DisplayName="gRPC Client Base Type"
```

Comment:
Given that this is an experimental feature, should we add this?

Reply:
can we mark it as experimental in the Description? If it still works without the enum definition, it's also fine to omit here. I'll leave the final decision to you, I don't feel strongly about this.

Reply:
It will work without the enum definition, but the users will need to edit it manually instead of being able to do it from the properties window in VS. I'll add an experimental blurb in the description.